packages/dcos-metrics-collector/build

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
The collector needs sufficient credentials to poll the local mesos agent. When being run manually they're configured via either `--auth_credential` or `AUTH_CREDENTIAL` envvar. Is it feasible to add that here? (preconfigured service account?)